SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1044,6 +1044,45 @@ void USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConne
 	// SpatialGDK - Here Unreal would return the position of the last replicated actor in PriorityActors before the channel became saturated.
 	// In Spatial we use ActorReplicationRateLimit and EntityCreationRateLimit to limit replication so this return value is not relevant.
 }
+
+void USpatialNetDriver::ProcessRPC(AActor* Actor, UObject* SubObject, UFunction* Function, void* Parameters)
+{
+	// The RPC might have been called by an actor directly, or by a subobject on that actor
+	UObject* CallingObject = SubObject != nullptr ? SubObject : Actor;
+
+	if (IsServer())
+	{
+		// Creating channel to ensure that object will be resolvable
+		if (GetOrCreateSpatialActorChannel(CallingObject) == nullptr)
+		{
+			UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Failed to get or create SpatialActorChannel for %s"), *CallingObject->GetName());
```

Comment:
What conditions does this happen in?

Reply:
That sounds like a failure in the function above.

Reply:
The only condition I know of is when EntityPool cannot provide a new id.

Reply:
We can also run out of channels. This could also happen in native Unreal, and they just drop the RPC in that case.
We should probably return here.

Reply:
In which case I am more certain that the error should come from inside that function. As that is where the failure is and it has context on what might have happened. Is behaviour from this point well defined?

Reply:
How many channels can you have? That sounds like serious error.

Reply:
As far as I can tell the behaviour is well definied from here. If we are anticipating well defined failures from here then the create function should start with try and we should be checking the output.

Reply:
I Agree, will remove this check.

Reply:
Removed.